austrakka/components/project/setup/funcs.py

```diff
@@ -0,0 +1,119 @@
+from typing import List
+from loguru import logger
+import pandas as pd
+from austrakka.utils.api import api_patch, api_get
+from austrakka.utils.misc import logger_wraps
+from austrakka.utils.output import print_table
+from austrakka.utils.paths import PROJECT_PATH
+
+# pylint: disable=duplicate-code
+@logger_wraps()
+def add_field_project(abbrev: str, field_names: List[str]):
+    field_name_objs = {"fieldAndSourceDTOs": []}
+    for field_name in field_names:
+        field_name_split = field_name.split(',')
+        field_name_objs["fieldAndSourceDTOs"].append({
+            "fieldName": field_name_split[0],
+            "restrictionName": field_name_split[1].replace('-', ' ')
+        })
+
+    # Replace 'abbrev' with your actual value
+    return api_patch(
+        path=f'{PROJECT_PATH}/add-project-field/{abbrev}',
+        data=field_name_objs
+    )
+
+
+@logger_wraps()
+def set_merge_algorithm_project(abbrev: str, merge_algorithm: str):
+    if merge_algorithm == 'show-all':
```

Comment:
Why not set `opt_merge_algorithm` `click.Choice` options to `ShowAll` and `Override` so we don't need to change what's passed in

Reply:
I think the lower case and dash notation suits the current language (other command sets) more than capital-case. In the interest of merging this with the least changes possible, lets go with this for now. We can always change it later.